bazel/ray.bzl

```diff
@@ -140,3 +140,55 @@ def copy_to_workspace(name, srcs, dstdir = ""):
         ),
         local = 1,
     )
+
+def native_java_binary(module_name, name, native_binary_name):
```

Comment:
I think Bazel has lots of Java rules -- could you please check to see if you can use those? If this isn't possible, please leave a comment explaining why they aren't sufficient, but I would be surprised if you can't leverage some of them. https://docs.bazel.build/versions/master/be/java.html

Reply:
The purpose of `native_java_binary` is to ensure a specific layout whcih is related out java code, see `io.ray.runtime.util.BinaryFileUtil#getNativeFile`. Existing java bazel rules is not for this